### PR TITLE
Add logging support and tests

### LINKED ISSUE #7: Add logging setup ala gitreload

### PR DESCRIPTION
**Pull request by carsongee**
Closes #7

### CODE REVIEW

**Comment by carsongee** [comment]
This has been updated to use our new flask-log extension, and should be quite a bit easier to evaluate

File: lmod_proxy/cmd.py [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
 
-from lmod_proxy import web
-
-log = logging.getLogger('lmod_proxy')
+from lmod_proxy import config
 
 
 def run_server():
-    logging.basicConfig(level=logging.DEBUG)
+
```
Comment (bdero) [inline lmod_proxy/cmd.py]: nit: Though it's not a requirement of pep8/pyflakes, want to add a docstring here?

Comment (carsongee) [inline lmod_proxy/cmd.py]: sure, it doesn't have any tests either, since this is just for debug running, but I'll at least add a doc string

Comment (carsongee) [inline lmod_proxy/cmd.py]: Added and filed issue for adding a test/no covering: #24

Comment (bdero) [inline lmod_proxy/cmd.py]: Yeah, that seems fine to me.

**Comment by bdero** [comment]
:sunny:

**Pull request merged**